addons/l10n_bh/data/template/account.fiscal.position-bh.csv

```diff
@@ -0,0 +1,4 @@
+"id","sequence","name","auto_apply","vat_required","country_id","country_group_id","tax_ids/tax_src_id","tax_ids/tax_dest_id","name@ar_001"
+"fiscal_position_template_dom","1","Bahrain","1","","base.bh","","","","البحرين"
+"fiscal_position_others","2","Non-Bahrain","1","","","l10n_bh.gulf_cooperation_council_without_bh","l10n_bh_purchase_vat_10","l10n_bh_purchase_vat_10_ex","غير البحرين"
```

Comment:
Seems weird we don't have an other fiscal position for non GCC countries. Currently an invoice to a US company uses the 10% tax and not the Export tax

And the fiscal position named Non-Bahrain is in fact GCC countries? (and does not use the tax named 0% GCC?)
Seems suspicious (or badly named but as a user that's not what I would expect)